go-controller/pkg/ovn/policy.go

```diff
@@ -414,33 +527,85 @@ func getL3MatchFromAddressSet(addressSets []string, policyType string) string {
 	return l3Match
 }
 
+func getMatchFromIPBlock(ipBlockCidr, portMatch, l4Match,
```

Comment:
Would you mind changing "portMatch" to "lportMatch".  When I read, portMatch can mean tcp and udp ports and that causes confusions.

Reply:
Done